### DISCUSSION TITLE
Uncertainty wedges for principal strain directions

### DISCUSSION THREAD

**Discussion by ncperdomo** [GenericMappingTools/gmt#8551] [Feature development discussion]
Hi GMT Team,

It would be great to add a feature to plot wedges showing 2-standard-deviation confidence intervals of the azimuth of the larger principal strain direction, similar to Figure 8 in [this paper](https://doi.org/10.1029/2023JB027472).

Thank you for considering this request!
Cheers,
N

![kj_strain_uncert_wedges](https://github.com/user-attachments/assets/4ba768f1-5fac-4d07-9a68-278baa709ffe)

**Comment by joa-quim** [comment]
Nicolás, to be very honest this feature request will hardly happen unless some interested at least starts to try implementing it. A possible, no C knowledge required, alternative is to try to create a custom symbol to that purpose.
